tests-web/assets/js/main.js

```diff
@@ -1,22 +1,39 @@
 import { makeDropHandler, makeLinkHandler, handleDragOver } from "./files.js";
+import { makeOpenTest } from "./tests.js";
 
-const cb = (map) => {
+const useMap = (map) => {
     const container = map.getContainer();
     container.ondrop = makeDropHandler(map);
     container.ondragover = handleDragOver;
-    console.info("Ready for drops on the map!", container);
 
     map.loadLink = makeLinkHandler(map);
+
+    console.info("New map created! File drops enabled.", container);
+
+    console.info("opening a test, just for fun...");
+    makeOpenTest(map)("aurora_sausage_link");
 }
 
 // Smuggle a reference to the created map, so I can work with it in JS land.
+console.debug("Listening in on map creations...");
 const LM = L.Map;
 window.maps = [];
 L.Map = function(x, opts = {
     maxZoom: 21,
 }, ...args) {
     const m = new LM(x, opts, ...args)
     window.maps.push(m);
-    setTimeout(() => cb(m), 0);
+    setTimeout(() => useMap(m), 0);
     return m;
 }
+
+// Settings for tile layers.
+const LL = L.TileLayer;
+L.TileLayer = function(x, opts = {
+    maxNativeZoom: 18,
+    maxZoom: 21,
+}, ...args) {
+    const l = new LL(x, opts, ...args)
```

Comment:
JS style question: any reason to have an intermediate variable and not just return?

Reply:
There is no functional difference, so there is no reason to introduce the local variable. This is just a bit of mess (I missed the `;` also). **I might put in `prettier` as a dev dependency to autoformat the source.**

This is like my [butter knife on the sink](https://img.ifunny.co/images/8e40ad8758ef62b3bffaff8e0ea716d224096b060124b0c8d2ddea7994594a1d_1.jpg); "maybe I'll use `l` again, but I'm not sure yet".